src/cli/cli_tcat.cpp

```diff
@@ -395,35 +398,102 @@ template <> otError Tcat::Process<Cmd("start")>(Arg aArgs[])
  * Done
  * @endcode
  * @par
- * Stops TCAT operation.
+ * Stops TCAT agent and stops TCAT operation.
  * @sa otBleSecureStop
  */
 template <> otError Tcat::Process<Cmd("stop")>(Arg aArgs[])
 {
-    OT_UNUSED_VARIABLE(aArgs);
+    otError error = OT_ERROR_NONE;
 
+    VerifyOrExit(aArgs[0].IsEmpty(), error = OT_ERROR_INVALID_ARGS);
     otBleSecureStop(GetInstancePtr());
 
-    return OT_ERROR_NONE;
+exit:
+    return error;
+}
+
+/**
+ * @cli tcat standby
+ * @code
+ * tcat standby
+ * Done
+ * @endcode
+ * @par
+ * Sets TCAT operation to standby, keeping the agent enabled.
+ * @sa otBleSecureTcatActive
+ */
+template <> otError Tcat::Process<Cmd("standby")>(Arg aArgs[])
+{
+    otError error;
+
+    VerifyOrExit(aArgs[0].IsEmpty(), error = OT_ERROR_INVALID_ARGS);
+    error = otBleSecureSetTcatAgentState(GetInstancePtr(), false, 0, 0);
+
+exit:
+    return error;
+}
+
+/**
+ * @cli tcat active
+ * @code
+ * tcat active
+ * Done
+ * @endcode
+ * @code
+ * tcat active 5000
+ * Done
+ * @endcode
+ * @code
+ * tcat active 5000 10000
+ * Done
+ * @endcode
+ * @par
```

Comment:
Please add a `@cparam` line after the code examples, to note the optional parameters.  https://openthread.io/reference/cli/contribute#parameters